README.md

```diff
@@ -737,6 +741,7 @@ These colorschemes may not specialize in Tree-sitter directly but are written in
 ### Web development
 
 - [NTBBloodbath/rest.nvim](https://github.com/NTBBloodbath/rest.nvim) - A fast Neovim http client written in Lua.
+- [ray-x/web-tools.nvim](https://github.com/ray-x/web-tools.nvim) - Launch a local development server with live reload feature for static & dynamic pages, http&ccs tag rename with LSP.
```

Comment:
http&css should be written as HTTP & CSS.

Reply:
thanks, it was fixed.